### PR TITLE
Remove owner-get hook tool.

### PR DESCRIPTION
**Pull request by cmars**
(Review request: http://reviews.vapour.ws/r/2280/)

### CODE REVIEW

File: worker/uniter/runner/jujuc/owner-get.go [hunk 1]
```diff
@@ -1,55 +0,0 @@
...
-	c.Key = args[0]
-	return cmd.CheckEmpty(args[1:])
-}
-
-func (c *OwnerGetCommand) Run(ctx *cmd.Context) error {
-	if c.Key != "tag" {
-		return fmt.Errorf("%s not set", c.Key)
-	}
-
-	return c.out.Write(ctx, c.ctx.OwnerTag())
```
Comment (mattyw) [inline worker/uniter/runner/jujuc/owner-get.go]: The call to ctx.OwnerTag here suggests a whole other series of things that can be removed. grep -rin "ownertag" ./worker. I'd be happy with this stuff being removed as well. 

There is lots of owner stuff inside the state server. We can leave that for now. but we could remove all traces of it from the uniter

Comment (cmars) [inline worker/uniter/runner/jujuc/owner-get.go]: Done.

**Comment by cmars** [comment]
@mattyw @fwereade PTAL

**Comment by cmars** [comment]
$$merge$$

**Pull request merged**